base/docs/helpdb.jl

```diff
@@ -16340,3 +16340,14 @@ If ``pkg`` is installed, return the installed version number, otherwise return `
 """
 Pkg.installed
 
+doc"""
+```rst
+::
+           randjump(r::MersenneTwister, jumps, [jumppoly]) -> Vector{MersenneTwister}
+
+Create an array of the size ``jumps`` of initialized ``MersenneTwister`` RNG objects where the first RNG object given as a parameter and following ``MersenneTwister`` RNGs in the array initialized such that a state of the RNG object in the array would be moved forward (without generating numbers) from a previous RNG object array element on a particular number of steps encoded by the jump polynomial ``jumppoly``.
```

Comment:
My brains has a hard time grasping this very very long sentence with no punctuation!